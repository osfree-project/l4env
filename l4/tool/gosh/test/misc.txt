
 TITLE TEXT
 can _be_ multi-line

 Authors

; $Id$

; This document demonstrates GOSH (Gold Out of SHit) and all its implemented
; features. It will be enhanced when GOSH is enhanced <--- no advice - simple
; rule!
; BTW: Comments start with ';'.

; FIXMEs / Questions
;
; * Do NOT strip all newlines - there could be intention to use 'em.
; * What is 'EMPTY'?
; * Will "LaTeX" be factored out of GOSH like HTML?

\tableofcontents

Intro
#####

Howto build this file:


!# cat gosh.txt > out.txt
!# sed s/^/\!/ gosh.txt >> out.txt
!# gosh out.txt > gosh.tex
!# rm out.txt


*Some hidden chapter
####################

1st Order Heading -- Chapter
############################

2nd Order Haeding -- Section
============================

3rd Order Heading -- SubSection
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

4th Order Heading -- Paragraph
------------------------------


Text styles
###########

Plain text looks like this.

Paragraph 1 is here.

; comments between paragraphs are not a problem 

Paragraph 2 should be here.

Use 'apostrophes for monospaced text', but #hash# works too for compatibility.

Trigger bug in newline handling:

BUG appears if a new line starts with non-words.

BUG this should be a new paragraph, but it is'nt.
(because of the bracket)

_Just_emphasize_this_point_ but only this point.

Trigger bug in emph: (_BUG_) - *GOTCHA* This bug is no more.
What about this one: (_BUG0_,_BUG1_)?

* itemize is nice.
* what do you think?
  * Yes?
  * No?
  * Maybe!

# Let's enumerate some points.
# Test point


:Description: Use this "for descriptions."
  The "description" continues here...

  and here.

:Advice: Watch the step!


Trigger bug in itemize newline handling:

BUG paragraph one

BUG paragraph two

BUG paragraph three
* item 1
* item 2
BUG paragraph four


!Whole line is verbatim. Maybe used for sources ...
!/** MAIN */
!void main(void) {
!    exit(0);
!}


"Abstract"
##########

_TODO_


References
##########

Ref section A
=============

Forward refer to section [Ref section B].


Ref section B
=============

Refer to section [Ref section A] and trigger _linebreak_bug_ [Ref
section A].

_TODO:_ bibs, figures, ...


Tables
######

Some Text could be here. Maybe more ...


 col1 | col2 | col3
 ------------------
 col1 | col2 | col3
 col1 | col2 | col3
 col1 | col2 | col3
 col1 | col2 | col3
 ------------------
 ------------------
 col1 | col2 | col3
[table LABEL] TEXT

 col1 | col2 | col3
 ------------------
 col1 | col2 | col3
[table TABLE2] second table

 accentuation  | test
----------------------------
 italic        | _italic_
 bold          | *bold*
 monospaced    | 'monospaced'

[table accentuations full-span]

Please not that you need a _space_ in the line between table [LABEL] and 
[TABLE2] in the source representation. Table [accentuations] shows
italic, bold, and monospaced styles.
;Is it correct that this text is indented?

More Text for Tab. [LABEL] follows here.


Images
######

_TODO_

;; usage as follows
;; <image_file_without_extension> will be label with some things
;; stripped of (e.g. '_')
;[image <image_file_without_extension> <scale>% <angle>°]
;caption
;more caption


"Bibliography"
##############

_TODO_


Source file of this document
############################

A damn url [http://www.hilton.de/dispatch/ViewPropertyHomePageForm?hid=11036375&ViewPropertyHomePage=&formName=ViewPropertyHomePageForm - Hilton]

Underlines in Latex citations text [xaz_123] text

There was a bug where an
'apostroph' caused the start of a new paragraph.

There was also a bug where a
[reference] caused the start of a new paragraph.

There was even a bug where a
"quotes" caused the start of a new paragraph.

Just to test, how -- and --- are handled by the backend.
BTW, what happens when they appear inside [refer --- ences]?


[http://a.link.as.section.title]
================================

Test for monospaced text within quotes
######################################

here comes "'some quoted monospaced text'" does it work?

here comes "#some quoted monospaced text#" with a hash?

here comes "_some_quoted italic text_"

here comes "*some quoted bold text*" followed by some normal text.

Text after image without empty line
###################################

'bla.png' is missing.
;[image bla.png]
This is some plain text that was wrongly assigned to the
caption of the image.

Accentuations
#############

Monospaced and 'non'-monospaced connected by a hyphen.

Accentuation of 'IPCmon''s name.

This works
##########

* _To be written by #Krishna#_
* _To be written by *Krishna* _
* _To be written by 'Krishna' _

This don't
##########

* _To be written by *Krishna*_
* _To be written by 'Krishna'_

Annotations
###########

:description:
  Here is some normal text
  | followed by an annotation
  and some more text
