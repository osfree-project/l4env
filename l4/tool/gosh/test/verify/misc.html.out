<html><head><title>TITLE TEXT can _be_ multi-line</title></head>
<body>
<h1><center>TITLE TEXT can _be_ multi-line</center></h1>
<b><center>Authors</center></b>
<center>2005-08-09</center>
<hr/><br/>
 <a name="Intro"></a>
 <h1>Intro</h1>
  <p>
   Howto build this file:
  </p>
<pre>
# cat gosh.txt > out.txt
# sed s/^/\!/ gosh.txt >> out.txt
# gosh out.txt > gosh.tex
# rm out.txt
</pre>
 <a name="1st_Order_Heading_--_Chapter"></a>
 <h1>1st Order Heading &ndash; Chapter</h1>
  <a name="2nd_Order_Haeding_--_Section"></a>
  <h2>2nd Order Haeding &ndash; Section</h2>
   <a name="3rd_Order_Heading_--_SubSection"></a>
   <h3>3rd Order Heading &ndash; SubSection</h3>
    <a name="4th_Order_Heading_--_Paragraph"></a>
    <h4>4th Order Heading &ndash; Paragraph</h4>
 <a name="Text_styles"></a>
 <h1>Text styles</h1>
  <p>
   Plain text looks like this.
  </p>
  <p>
   Paragraph 1 is here.
  </p>
  <p>
   Paragraph 2 should be here.
  </p>
  <p>
   Use <code>apostrophes for monospaced text</code>, but <code>hash</code> works too for compatibility.
  </p>
  <p>
   Trigger bug in newline handling:
  </p>
  <p>
   BUG appears if a new line starts with non-words.
  </p>
  <p>
   BUG this should be a new paragraph, but it is'nt.
   (because of the bracket)
  </p>
  <p>
   <i>Just emphasize this point</i> but only this point.
  </p>
  <p>
   Trigger bug in emph: (<i>BUG</i>) - <b>GOTCHA</b> This bug is no more.
  </p>
  <ul>
   <li>
    <p>
     itemize is nice.
    </p>
   </li>
   <li>
    <p>
     what do you think?
    </p>
    <ul>
     <li>
      <p>
       Yes?
      </p>
     </li>
     <li>
      <p>
       No?
      </p>
     </li>
     <li>
      <p>
       Maybe!
      </p>
     </li>
    </ul>
   </li>
  </ul>
  <ol>
   <li>
    <p>
     Let's enumerate some points.
    </p>
   </li>
   <li>
    <p>
     Test point
    </p>
   </li>
  </ol>
  <dl>
   <dt>Description</dt>
   <dd>
    <p>
     Use this "for descriptions."
     The "description" continues here...
    </p>
    <p>
     and here.
    </p>
   </dd>
   <dt>Advice</dt>
   <dd>
    <p>
     Watch the step!
    </p>
   </dd>
  </dl>
  <p>
   Trigger bug in itemize newline handling:
  </p>
  <p>
   BUG paragraph one
  </p>
  <p>
   BUG paragraph two
  </p>
  <p>
   BUG paragraph three
  </p>
  <ul>
   <li>
    <p>
     item 1
    </p>
   </li>
   <li>
    <p>
     item 2
    </p>
   </li>
  </ul>
  <p>
   BUG paragraph four
  </p>
<pre>
Whole line is verbatim. Maybe used for sources ...
/** MAIN */
void main(void) {
    exit(0);
}
</pre>
 <a name="Abstract"></a>
 <h1>"Abstract"</h1>
  <p>
   <i>TODO</i>
  </p>
 <a name="References"></a>
 <h1>References</h1>
  <a name="Ref_section_A"></a>
  <h2>Ref section A</h2>
   <p>
    Forward refer to section <a href="#Ref_section_B">Ref section B</a>.
   </p>
  <a name="Ref_section_B"></a>
  <h2>Ref section B</h2>
   <p>
    Refer to section <a href="#Ref_section_A">Ref section A</a> and trigger <i>linebreak bug</i> [Ref
    section A].
   </p>
   <p>
    <i>TODO:</i> bibs, figures, ...
   </p>
 <a name="Tables"></a>
 <h1>Tables</h1>
  <p>
   Some Text could be here. Maybe more ...
  </p>
  <a name="LABEL"></a>
  <br/><center><table cellspacing="0" cellpadding="3" border="1">
  <tr>  <th bgcolor=#f0f7ff> col1 <br/></th>  <th bgcolor=#f0f7ff> col2 <br/></th>  <th bgcolor=#f0f7ff> col3<br/></th>  </tr>
  <tr>
  <td> col1</td>
  <td> col2</td>
  <td> col3</td>
  </tr>
  <tr>
  <td> col1</td>
  <td> col2</td>
  <td> col3</td>
  </tr>
  <tr>
  <td> col1</td>
  <td> col2</td>
  <td> col3</td>
  </tr>
  <tr>
  <td> col1</td>
  <td> col2</td>
  <td> col3</td>
  </tr>
  <tr>
  <td> col1</td>
  <td> col2</td>
  <td> col3</td>
  </tr>
  </table>
  Table 1: TEXT
  </center><br/>
  <p>
   More Text for Tab. <a href="#LABEL">1</a> follows here.
  </p>
 <a name="Images"></a>
 <h1>Images</h1>
  <p>
   <i>TODO</i>
  </p>
 <a name="Bibliography"></a>
 <h1>"Bibliography"</h1>
  <p>
   <i>TODO</i>
  </p>
 <a name="Source_file_of_this_document"></a>
 <h1>Source file of this document</h1>
  <p>
   A damn url <a href="http://www.hilton.de/dispatch/ViewPropertyHomePageForm?hid=11036375&ViewPropertyHomePage=&formName=ViewPropertyHomePageForm" alt="Hilton">Hilton</a>
  </p>
  <p>
   Underlines in Latex citations text <a href="xaz_123" alt="xaz_123">xaz_123</a> text
  </p>
  <p>
   There was a bug where an
   <code>apostroph</code> caused the start of a new paragraph.
  </p>
  <p>
   There was also a bug where a
   <a href="reference" alt="reference">reference</a> caused the start of a new paragraph.
  </p>
  <p>
   There was even a bug where a
   "quotes" caused the start of a new paragraph.
  </p>
  <p>
   Just to test, how &ndash; and &mdash; are handled by the backend.
   BTW, what happens when they appear inside <a href="refer --- ences" alt="refer --- ences">refer &mdash; ences</a>?
  </p>
  <a name="[http://a.link.as.section.title]"></a>
  <h2><a href="http://a.link.as.section.title" alt="http://a.link.as.section.title">http://a.link.as.section.title</a></h2>
 <a name="Test_for_monospaced_text_within_quotes"></a>
 <h1>Test for monospaced text within quotes</h1>
  <p>
   here comes "<code>some quoted monospaced text</code>" does it work?
  </p>
  <p>
   here comes "<code>some quoted monospaced text</code>" with a hash?
  </p>
  <p>
   here comes "<i>some quoted italic text</i>"
  </p>
  <p>
   here comes "<b>some quoted bold text</b>" followed by some normal text.
  </p>
 <a name="Text_after_image_without_empty_line"></a>
 <h1>Text after image without empty line</h1>
  <table cellspacing="10"><tr><td>
    <img src="bla.png.jpg" border="0">
  </td></tr></table>
  <p>
   This is some plain text that was wrongly assigned to the
   caption of the image.
  </p>
 <a name="Accentuations"></a>
 <h1>Accentuations</h1>
  <p>
   Monospaced and 'non'-monospaced connected by a hyphen.
  </p>
<hr/><br/>
<i>Authors</i>
</body>
</html>
